src/contexts/RequestContext/RequestCanvasToolbar.test.tsx

```
@@ -0,0 +1,287 @@
+      url: 'https://api.example.com/users',
+      method: 'GET',
+      headers: {},
+      body: '',
+      response: null,
+      isLoading: false,
+    });
+
+    useHistoryStore.setState({
+      entries: [],
+    });
+  });
```

Biome (2.3.14)

[error] Duplicate beforeEach hook found.

Remove this duplicate hook or consolidate the logic into a single hook.

(lint/suspicious/noDuplicateTestHooks)